instrumentation/opentelemetry-instrumentation-openai/src/opentelemetry/instrumentation/openai/patch.py

```diff
@@ -0,0 +1,308 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from opentelemetry import trace
+from opentelemetry.trace import SpanKind, Span
+from opentelemetry.trace.status import Status, StatusCode
+from opentelemetry.trace.propagation import set_span_in_context
+from .span_attributes import LLMSpanAttributes
+from opentelemetry.semconv._incubating.attributes import (
+    gen_ai_attributes as GenAIAttributes,
+)
+from .utils import (
+    silently_fail,
+    extract_content,
+    get_llm_request_attributes,
+    is_streaming,
+    set_span_attribute,
+    set_event_completion,
+    extract_tools_prompt,
+)
+from opentelemetry.trace import Tracer
+
+
+def chat_completions_create(tracer: Tracer):
+    """Wrap the `create` method of the `ChatCompletion` class to trace it."""
+
+    def traced_method(wrapped, instance, args, kwargs):
+
+        llm_prompts = []
+
+        for item in kwargs.get("messages", []):
+            tools_prompt = extract_tools_prompt(item)
+            llm_prompts.append(tools_prompt if tools_prompt else item)
+
+        span_attributes = {
+            **get_llm_request_attributes(kwargs, prompts=llm_prompts),
+        }
+
+        attributes = LLMSpanAttributes(**span_attributes)
+
+        span_name = f"{attributes.gen_ai_operation_name} {attributes.gen_ai_request_model}"
+
+        span = tracer.start_span(
+            name=span_name,
+            kind=SpanKind.CLIENT,
+            context=set_span_in_context(trace.get_current_span()),
```

Comment:
there should be no need, this is the default behavior
```suggestion
```

Reply:
agree, will remove